README.md

```diff
@@ -1 +1,253 @@
-# vapor
\ No newline at end of file
+![Vapor Logo](docs/logo.png)
+
+# Vapors
+
+The Vapors library provides an embedded-DSL for writing logical queries over a collection of facts.
+It is based on the `cats-free` implementation of `FreeApplicative` for building the underlying algebra
+into an expression that can be optimized, evaluated, serialized, and interpreted in different ways.
+
+## Getting Started
+
+<details>
+
+### Setup
+
+1.  **Add it to your `build.sbt`**
+
+    ```
+    resolvers += Resolver.bintray("jeffmay", "maven")
+    libraryDependencies += "com.rallyhealth" %% "vapors" % "0.1.0"
+    ```
+
+2.  **Define your fact types.**
+
+    Every fact type must have a unique name as well as a Scala type. You can define a FactType with any Scala type you
+    want, but if you want to be able to use comparison operators, you must define an `Ordering`. To check equality, you
+    need to define `cats.Eq`.
+
+    ```
+    object FactTypes {
+      val DateOfBirth = FactType[LocalDate]("DateOfBirth")
+      val Role = FactType[String]("Role")
```

Comment:
`String` should be `Role`. Also, I think you might need to rename `Role` because of shadowing. Also, the code below needs to use `Role`.